code/player/Inventory.cs

```diff
@@ -37,7 +34,7 @@ public override void DeleteContents()
             RPCs.ClientClearInventory(To.Multiple(Utils.GetClients((pl) => pl.CurrentPlayer == Owner as TTTPlayer)));
 
             Perks.Clear();
-            Ammo.Clear();
+            (Owner as TTTPlayer).Ammo.Clear();
```

Comment:
Why saving ammo on playaer instead on inventory now?

Reply:
Since SWB stores ammo on the player.